### DISCUSSION TITLE
Google Addsense

### DISCUSSION THREAD

**Discussion by thelocalh0st** [cotes2020/jekyll-theme-chirpy#1181] [Q&A]
How to add google addsense to the site!!

**Comment by sublimeclemency** [comment]
@thelocalh0st these commits from https://github.com/otzslayer/otzslayer.github.io/commits could help:
https://github.com/otzslayer/otzslayer.github.io/commit/515482f76e564d34ac83ac940b17eef69a52d830
https://github.com/otzslayer/otzslayer.github.io/commit/966503b87b8255c48ba02d52200493bb9b570f99
https://github.com/otzslayer/otzslayer.github.io/commit/30ef44199d45fe8d002597cffebdca042633b2a7
https://github.com/otzslayer/otzslayer.github.io/commit/d32e86ab74a42e43489d714b025f33d7034df72b
https://github.com/otzslayer/otzslayer.github.io/commit/44c8977dba1e719901a441e4395d25dd86b0bb02
https://github.com/otzslayer/otzslayer.github.io/commit/76da95199e5ec3f97229e4008b871668ea63a711

i am bad at using jekyll too 👀

**Comment by sujayadkesar** [comment]
Thanks for the reference!! ll checkout! 🤝
